sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java

```diff
@@ -90,4 +92,12 @@ public String getProviderName() {
     public boolean verify(String encryptionKeyId, boolean allowEnclaveComputations, byte[] signature) throws MicrosoftDataEncryptionException {
         throw new MicrosoftDataEncryptionException("The Verify operation is not supported. ");
     }
+
+    private static String getNameForKeyEncryptionKeyAlgorithm(KeyEncryptionKeyAlgorithm keyEncryptionKeyAlgorithm) {
+        if(keyEncryptionKeyAlgorithm == KeyEncryptionKeyAlgorithm.RSA_OAEP) {
+            return "RSA-OAEP";
```

Comment:
I think this can be the value property of a enum

Reply:
So AAP lib is only using enum with this value , for now we will go with this hard coding and in samples after I will expose key algo on encrytionKeyWrapMetadata in my next PR , we will show https://github.com/Azure/azure-sdk-for-java/blob/3f31d68eed6fbe11516ca3afe3955c8840a6e974/sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/models/KeyWrapAlgorithm.java#L20

Also we are in conversation with AAP dev to use string instead of enum so we will just pass through the string from AAP cryptography to generic KeyEncryptionKeyResolver